### ISSUE TITLE
geojson Points can't be visualized as Kepler point layer (only polygon)

### ISSUE THREAD

**Issue by stephanietuerk** [keplergl/kepler.gl#177]
First: This is an amazing project! Thank you!

Issue: When I import a geojson file consisting of Point geometry types, Kepler defaults to visualizing this with a polygon layer. If I try to switch the layer to point (so that I can use visualization techniques like hexbin), I can't get it to read the lat/lon coordinates of the Point geometries/display the points. 
Selecting value _geojson in Lat and Lng boxes does not make points appear in map.

I can resolve this problem by transforming geojson back into regular json or csv by extracting lat/lon coords from geometry (using Python), but if geojsons are supported, would be nice for Point geometries to display properly.

MacBook Pro Retina 13-inch early 2015
OS: 10.13.5
Chrome Version 67.0.3396.99 (Official Build) (64-bit)

geojson snippet:
{
"type": "FeatureCollection",
"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
"features": [
{ "type": "Feature", "properties": { "geo_precision": 1, "count": 1 }, "geometry": { "type": "Point", "coordinates": [ 60.8944, 32.2632 ] } },
{ "type": "Feature", "properties": { "geo_precision": 1, "count": 1 }, "geometry": { "type": "Point", "coordinates": [ 61.0667, 34.6667 ] } },
{ "type": "Feature", "properties": { "geo_precision": 1, "count": 2 }, "geometry": { "type": "Point", "coordinates": [ 61.23, 31.8261 ] } },
....

**Comment by sashuk** [comment]
First, thanks a lot for such a great tool! :fireworks: 

Concerning the issue: the line objects (GeoJSON layers) are not detected correctly as well, the example below is detected as the polygon layer.

```
{
	"type": "FeatureCollection",
	"features": [{
		"geometry": {
			"type": "LineString",
			"coordinates": [
				[39.248915, -6.961785],
				[39.250374, -6.952925]
			]
		},
		"type": "Feature",
		"properties": {
			"gid": 81,
			"id": null
		}
	}]
}
```

**Comment by prune998** [comment]
same issue here. Any info on this ? Is it sa bug, a lack of feature or we're doing something wrong ?

**Comment by stephanietuerk** [comment]
> Any info on this ? Is it sa bug, a lack of feature or we're doing something wrong ?

Hey prune998 -- I'm now using deck.gl, which viz library kepler was written on top of, which I think gives me some insight in the the issue that might be helpful as we await an official answer. 

In deck.gl, all geojson data, whether Points or Polygons or MultiPolygons, etc. is handled by a geojson layer. There is a separate ScatterPlot layer that can take two coordinates (x,y / lon/lat) from data that isn't formatted as a geojson and will by default visualize it with a circle marker. I personally think this is not an advantageous way to structure the deck layers -- they shouldn't have different layers that make the same visualization based on differences in data formatting, but -- they do.

My guess is that the Point layer in kepler is using the ScatterPlot layer from deck, which can't read geojson data. Which means that, no, we're not doing anything wrong.

**Comment by stephanietuerk** [comment]
Oh, if you simply convert your data to a .csv with either lon/lat or x,y  headers for the corresponding coordinates, you can then upload it into a Point layer in kepler. You can use something like [Mapshaper]([url](https://mapshaper.org/)) for this.

Going to deck.gl is a big step (fair amount of code to write) if you just need to visualize a dataset for your own purposes.

**Comment by stevepepple** [comment]
Thanks for the suggestion @stephanietuerk!
You shouldn't have to convert to csv though, the tools should properly handle GeoJSON feature types.

**Comment by heshan0131** [comment]
If you add `lat` `lng` to your geojson `feature.properties`, kepler.gl will create a point layer

**Comment by macrigiuseppe** [comment]
@stevepepple we are in the process to upgrade to deck.gl 6.4 and we will fix it

**Comment by heshan0131** [comment]
#507

**Comment by heshan0131** [comment]
Close as it's merged into `1.0.0`

**Comment by Flowm** [comment]
Sorry for bringing up this old topic, but for me loading geojson files with Point geometries still only shows the polygon layer and lat/lon coordinates aren't available for all other layers (e.g hexbin).

<img width="160" alt="image" src="https://user-images.githubusercontent.com/1117666/84851302-30c54000-b05a-11ea-9ecf-62df5a72efb8.png">

Is this something that that should work or is the geojson to csv workaround still required?

Sample geojson
```
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          -111.413358,
          33.793864
        ]
      },
      "properties": {
        "age": 2872,
        "accuracy": 3140
      }
    },
    {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          -111.411,
          33.755
        ]
      },
      "properties": {
        "age": 382,
        "accuracy": 1000
      }
    }
  ]
}
```
